css-two-column-layout/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div id="container">
+    <div class="row">
+      <div class="column-full">
+        <h1 class="column-third">
+          Scale, Speed, Agility
+        </h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full">
+        <div class="column-third">
+          <p>
+            We've dedicated thousands of man hours to making healthcare institutions in
+            the Helium <br>network run 10x more efficiently and deliver an unparalleled
+            quality of care.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-ten">
+        <div>
+          <img src="images/software.svg" alt="software" class="image">
+        </div>
+      </div>
+      <div class="column-thirty">
+        <div class="row">
+          <h2>
+            Intuitive, <br>Smart Software
+          </h2>
+        </div>
+        <div class="row">
+          <p>
+            We build produts that transform how healthcare facilities run across Africa - making them
+            radically more efficient operationally and financially, a complete turnaround.
+          </p>
+        </div>
+      </div>
```

Comment:
For this section, lines 33 to 50 I would recommend wrapping the `column-ten` and `column-thirty` in a column-fifty which is set to 50% flex-basis. Then you can use display flex on the `column-fifty` element to control the sub columns. That will give you the true two column layout